boot/zephyr/pm.yml

```diff
@@ -81,7 +81,12 @@ mcuboot_primary_1:
 
 #if (CONFIG_NRF53_MULTI_IMAGE_UPDATE)
 mcuboot_secondary_1:
+#if (CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY_1)
   region: external_flash
+#endif /* CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY_1 */
   size: CONFIG_NRF53_RAM_FLASH_SIZE
+  placement:
```

Comment:
We need to make sure that the alignment for internal flash is the same as for mcuboot primary. That should be page aligned. Also we need to make this block different between external flash and internal flash configurations.

Reply:
Do you know why the flash alignment in external flash is set to 4 by default then?
ref: https://github.com/nrfconnect/sdk-mcuboot/blob/ccc722742bf7d54350f0b8b59c8bead0426e5e56/boot/zephyr/pm.yml#L24

 This could potentially be a misconfiguration and it should be page aligned regardless. 
 Also should I look into potentially using
 ```
 zephyr/dts.cmake:set_target_properties(devicetree_target PROPERTIES "DT_REG|/soc/flash-controller@4001e000|SIZE" "0x1000;")
 ```
 
 For setting the alignment requirements so that it would be page aligned regardless of the device used.